src/Core/Mesh/Wrapper/Convert.cpp

```diff
@@ -132,6 +132,53 @@ void convert( const Dcel& dcel, TriangleMesh& mesh ) {
 }
 
 
+/// Particular conversion for a progressive mesh
```

Comment:
What is this code ?

Reply:
don't know, it was here from @Syn4ck, and I comment it ...

Reply:
`ProgressiveMesh` looks like something from Céline or Anahid's ?

Reply:
Yes ... do you prefer to remove the commented code ?

Reply:
Lets make sure this code isn't useful to anyone. If it is let them keep it somewhere else, then delete it. Let's not add dead code (its' already enough to maintain the living one :) )

Reply:
OK I delete it, if it is useful, it must have been somewhere else !